docker-compose.yaml

```diff
@@ -43,6 +45,8 @@ services:
       - POSTGRES_DB=keycloak
     volumes:
       - ./scripts/docker-compose/volume-data/zac-keycloak-database-data:/var/lib/postgresql/data
+    extra_hosts:
```

Comment:
Do we really need this everywhere? We only use `host.docker.internal` for one or two containers right?